app/models/spree/stock_location_decorator.rb

```diff
@@ -0,0 +1,14 @@
+Spree::StockLocation.class_eval do
+  def move(variant, quantity, originator = nil)
+    # if we are dealing with an override
+    if variant.@variant_override.andand.count_on_hand.nil?
```

Comment:
I don't understand this branch. If there is no variant override, we should go into this line as well, right?

Reply:
yes, that's broken and outdated. I committed on top of that, see "files changed". it's now
`if variant.is_override?`